node-util/sbin/oo-last-access

```diff
@@ -115,7 +121,8 @@ class LastAccessUpdater
     # Doing the file read with awk is 10x faster than doing it inside
     # Ruby and we desperately need speed here.
     ignore_cond = [@hosts_ignore.map { |a| "$2 != \"#{a}\"" },
-                   @addrs_ignore.map { |a| "$1 != \"#{a}\"" }].flatten.join(' && ')
+                   @addrs_ignore.map { |a| "$1 != \"#{a}\"" },
+                  "$_ !~ /#{ @useragent_ignore.join('|') }/" ].flatten.join(' && ')
```

Comment:
This will break if `UNIDLER_IGNORE_LIST` contains slashes, and User-Agent strings can (and usually do) include slashes.  This will also interpret special regex characters (".", "*", "+", "?", "(", ")", "[", "]", "|", "\").  We should escape `@useragent_ignore`.

Could someone create an application or domain that contained something in `UNIDLER_IGNORE_LIST`? We probably shouldn't use `$_`.